src/app/event/Modal.tsx

```diff
@@ -45,6 +45,8 @@ export function Modal({ eventId }: { eventId: string }) {
       };
     });
 
+    console.log("Attesting tickets", tickets);
```

Comment:
It is a must on the app or it's just for testing? If it's for testing and you will remove it later, you should add a TODO to remove it later